mPowerSDK/TasksAndSteps/APHActivityManager.m

```diff
@@ -136,7 +136,12 @@ + (instancetype)defaultManager
     }
     
     // Replace the language in the last step
-    [task.steps.lastObject setTitle:NSLocalizedStringWithDefaultValue(@"APH_ACTIVITY_CONCLUSION_TEXT", nil, APHLocaleBundle(), @"Thank You!", @"Main text shown to participant upon completion of an activity.")];
+    if ([surveyId isEqualToString:APHMemoryActivitySurveyIdentifier]) {
```

Comment:
I know the memory task is a little unusual in that the number of steps it shows you depends on how well you do--so I would have guessed maybe they did it this way because they didn't know in advance which step would be the conclusion? So just want to make sure, does it still show the right thing at the end if you do the tests "wrong"?